opm/simulators/linalg/bda/BdaBridge.cpp

```diff
@@ -121,7 +122,7 @@ void getSparsityPattern(BridgeMatrix& mat, std::vector<int> &h_rows, std::vector
 #endif
 
 template <class BridgeMatrix, class BridgeVector>
-void BdaBridge::solve_system(BridgeMatrix *mat OPM_UNUSED, BridgeVector &b OPM_UNUSED, InverseOperatorResult &res OPM_UNUSED)
+void BdaBridge::solve_system(BridgeMatrix *mat OPM_UNUSED, BridgeVector &b OPM_UNUSED, WellContributions& wellContribs OPM_UNUSED, InverseOperatorResult &res OPM_UNUSED)
```

Comment:
But if CUDA is not found by CMake, it is actually unused.

Reply:
It seems like it will never even be compiled in this case, see [here](/OPM/opm-simulators/blob/d067c08be7d91a0df380d96578a2d9487583891a/CMakeLists_files.cmake#L46-L48)

Reply:
The BdaBridge is always compiled, but only created when CUDA is found. I think BdaBridge should also be in the CUDA-only list. And ISTLSolverEbos should only include the header if CUDA is found.
We could also add more #if HAVE_CUDA guards in the well files, since they don't need WellContributions.hpp if CUDA is not found.

Reply:
Nevermind. I missed that is compiled unconditionally.